site/content/docs/install.md

```diff
@@ -10,7 +10,8 @@ date: 2017-05-07T15:04:10.000Z
 (Prerequisite) Platform Docker Install
 --------------------------------------
 
-> Ignore this step if you already have a current docker install. Go to [Install SNAS Using Docker](#install-snas-using-docker)
+> Ignore this step if you already have a current docker install. Follow instructions from the submenu on left to install
```

Comment:
IMO, removing the link is less interactive. See other comment about the fix @serpilb has.